### PR TITLE
bugfix: fix escaped characters for upper and lower case column names

### PR DESCRIPTION
**Pull request by a364176773**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
https://github.com/seata/seata/pull/5658

### CODE REVIEW

File: sqlparser/seata-sqlparser-core/src/main/java/io/seata/sqlparser/EscapeHandler.java [hunk 1]
```diff
@@ -61,14 +62,27 @@ default boolean containsEscape(String fieldOrTableName) {
...
+     * @param tableMeta tableMeta
+     * @return colName
+     */
+    default String addColNameEscape(String colName, TableMeta tableMeta) {
         boolean needEscape = checkIfNeedEscape(colName);
         if (!needEscape) {
             return colName;
         }
+        if (tableMeta != null) {
+            colName = tableMeta.getColumnMeta(colName).getColumnName();
```
Comment (slievrly) [inline sqlparser/seata-sqlparser-core/src/main/java/io/seata/sqlparser/EscapeHandler.java:84]: check the columnMeta if empty.

Comment (a364176773) [reply sqlparser/seata-sqlparser-core/src/main/java/io/seata/sqlparser/EscapeHandler.java]: done

**Review by slievrly — COMMENTED** [review]
TODO:  OracleEscapeHandler#checkIfNeedEscape 、 PostgresqlEscapeHandler#checkIfNeedEscape

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**